scripts/deploy/govcms-cache-rebuild

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+IFS=$'\n\t'
+set -euo pipefail
+
+#
+# GovCMS database backups.
+#
+# Allow a cache-rebuild to be toggled at deploy time.
+#
+
+LAGOON_ENVIRONMENT_TYPE=${LAGOON_ENVIRONMENT_TYPE:-production}
+GOVCMS_CACHE_REBUILD=${GOVCMS_CACHE_REBUILD:-true}
+
+echo "GovCMS Deploy :: Cache rebuild"
+
+if [ "$GOVCMS_CACHE_REBUILD" != "true" ]; then
```

Comment:
do we want the comparison to the string `"true"` or just to check if the value of the variable is set? Do we have an agreement on this standard across scripts?